setup/cmd/root.go

```diff
@@ -66,10 +68,11 @@ func Execute() {
 	cmd.Flags().IntVarP(&userBatches, "batch", "b", 25, "create user accounts in batches of N, increasing batch size may cause performance problems")
 	cmd.Flags().StringVar(&hostOperatorNamespace, "host-ns", defaultHostNS, "the namespace of Host operator")
 	cmd.Flags().StringVar(&memberOperatorNamespace, "member-ns", defaultMemberNS, "the namespace of the Member operator")
-	cmd.Flags().StringSliceVar(&templatePaths, "template", []string{}, "the path to the OpenShift template to apply")
+	cmd.Flags().StringSliceVar(&templatePaths, "template", []string{}, "the path to the OpenShift template to apply for each active user namespace")
 	cmd.Flags().IntVarP(&activeUsers, "active", "a", 3000, "how many users will have the user workloads template applied")
 	cmd.Flags().BoolVar(&skipCSVGen, "skip-csvgen", false, "if an all-namespaces operator should be installed to generate a CSV resource in each namespace")
 	cmd.Flags().BoolVar(&skipDefaultTemplate, "skip-default-template", false, "skip the setup/resources/user-workloads.yaml template file when creating resources")
+	cmd.Flags().IntVar(&numberOfOperators, "operators", len(operators.Templates), "configures number of sandbox operators to install. (defaults to all - order is not configurable)")
```

Comment:
Yeah, I must admit the param name and the wording is what I spent the least time on and I was struggling to word them. I was more focused on making it functional. 🙂  So I know there's definitely room for improvement.

1. Param name
For the actual name, how about `--operators-limit` instead of just `--operators`?

2. Description
What about something like `can be specified to limit the number of operators to install (by default all operators are installed to simulate cluster load in production)`

Reply:
That would work.
But let's add "additional" before "operators":

"... the number of additional operators to install..."

Reply:
Done https://github.com/codeready-toolchain/toolchain-e2e/pull/314/commits/924d02fee5e20d62154d1a1958371927ebc994a9